src/Middleware/ApmMiddleware.php

```diff
@@ -51,7 +53,9 @@ public function handle($request, Closure $next)
 
         $response = $next($request);
 
-        if (!$sampler->shouldSample()) {
+        $reject = $this->shouldBeRejected($request, $filters);
```

Comment:
No need for tmp variable here

Reply:
the style checker pr (#3 ) should help with that

Reply:
roger